solidity/contracts/Peggy.sol

```diff
@@ -131,18 +131,18 @@ contract Peggy {
 	) private pure {
 		uint256 cumulativePower = 0;
 
-		for (uint256 k = 0; k < _currentValidators.length; k++) {
+		for (uint256 i = 0; i < _currentValidators.length; i++) {
```

Comment:
what exactly is the purpose of using k or i in this case?

Reply:
just clearer to use i, k was being used because this code was once inside a nested loop